test/controllers/HomepageControllerSpec.scala

```diff
@@ -92,6 +95,19 @@ class HomepageControllerSpec extends FrontEndTestHelper {
       homepagePageAsString must not include viewTransferButton
     }
 
+    "render the DTA review homepage page with an authenticated tna user" in {
```

Comment:
Might want a test here to make sure a user with the incorrect authorisation can't access the TNA user homepage content.

Reply:
I've added lines in existing tests to check that the transfers for review button is not present for judgment or standard users, but will refactor out some sort of content method later on once the page becomes more substantial.

For no authorisation at all, we already seem to have a check to ensure redirects to the auth server from each of the paths we are using (`/judgment/homepage` and `/homepage`, which is currently used for the TNA homepage as well, though we will want to implement a dedicated route for TNA pages as they get added in further down the line).